ddtrace/contrib/celery/signals.py

```diff
@@ -129,6 +123,8 @@ def trace_failure(*args, **kwargs):
         ex = kwargs.get('einfo')
         if ex is None:
             return
+        if hasattr(task, 'throws') and isinstance(ex.exception, task.throws):
```

Comment:
Should this be `type(ex.exception) in task.throws`?

Or maybe a:

```python
for exc_type in task.throws:
    if isinstance(ex.exception, exc_type):
        return
```

?

Reply:
According to the docs, it seems like `task.throws` can be a list

Reply:
nvm, I am silly, `isinstance` accepts either a single type or a list of types... duh...